packages/components/src/__actions__/Menu/v3/_docs/Menu.spec.stories.tsx

```diff
@@ -0,0 +1,287 @@
+      await userEvent.click(menuButton)
+      await waitFor(() => expect(canvas.getByRole("menu")).toBeVisible())
+    })
+
+    await step("Arrow keys adjust focus", async () => {
+      await userEvent.keyboard("[ArrowDown]")
+      await waitFor(() =>
+        expect(
+          canvas.getByRole("menuitem", { name: "Go to cultureamp.com" })
+        ).toHaveFocus()
+      )
+    })
+
+    await step("Esc closes menu", async () => {
+      await userEvent.keyboard("[Escape]")
+      await waitFor(() =>
+        expect(canvas.queryByRole("menu")).not.toBeInTheDocument()
+      )
+    })
+
+    await step("Menu opens on enter press", async () => {
+      await userEvent.keyboard("[Enter]")
+      await waitFor(() =>
+        expect(
+          canvas.getByRole("menuitem", { name: "Trigger an alert" })
+        ).toHaveFocus()
+      )
+      await userEvent.keyboard("[Escape]")
+    })
+
+    await step("Menu opens on arrow up/down", async () => {
+      await userEvent.keyboard("[ArrowDown]")
+      await waitFor(() =>
+        expect(
+          canvas.getByRole("menuitem", { name: "Trigger an alert" })
+        ).toHaveFocus()
+      )
+      await userEvent.keyboard("[Escape]")
+
+      await userEvent.keyboard("[ArrowUp]")
+      await waitFor(() =>
+        expect(canvas.getByRole("menuitem", { name: "Item 5" })).toHaveFocus()
+      )
+    })
+  },
+}
+
+const mockOnClick = fn()
+export const DisabledItems: Story = {
+  render: () => (
+    <MenuTrigger>
+      {/* Replace with Kaizen Button once we have v3 or backwards compatibility */}
+      <RACButton className="bg-white border border-gray-500 rounded p-8 flex">
+        <MeatballsIcon role="img" aria-label="Additional actions" />
+      </RACButton>
+      <Popover>
+        <Menu>
+          <MenuItem isDisabled onAction={mockOnClick}>
+            Item 1
+          </MenuItem>
+          <MenuItem>Item 2</MenuItem>
+          <MenuItem isDisabled>Item 3</MenuItem>
+          <MenuItem>Item 4</MenuItem>
+          <MenuItem>Item 5</MenuItem>
+        </Menu>
+      </Popover>
+    </MenuTrigger>
+  ),
+  play: async ({ canvasElement, step }) => {
+    const canvas = within(canvasElement.parentElement!)
+    const menuButton = canvas.getByRole("button")
+
+    await step("Disabled items are unable to be focused", async () => {
+      await userEvent.click(menuButton)
+      expect(canvas.getByRole("menuitem", { name: "Item 2" })).toHaveFocus()
+      await userEvent.keyboard("[ArrowDown]")
+      await userEvent.keyboard("[ArrowDown]")
+      expect(canvas.getByRole("menuitem", { name: "Item 5" })).toHaveFocus()
+      await userEvent.keyboard("[Escape]")
+    })
+
+    await step("Clicking a disabled item doesn't trigger onClick", async () => {
+      await userEvent.click(menuButton)
+      await userEvent.click(canvas.getByRole("menuitem", { name: "Item 1" }))
+      await waitFor(() => expect(mockOnClick).not.toBeCalled())
+    })
+  },
+}
+
+export const WithSections: Story = {
```

Comment:
I think we'll also want to add a play story to have an open state chromatic can snapshot - If there's a story I missed that already capture this, feel free to disregard 🙇

Reply:
Missed the kitchen sink story! Happily disregard